cookbooks/aws-parallelcluster-slurm/files/default/config_slurm/scripts/health_checks/gpu_health_check.sh

```diff
@@ -0,0 +1,209 @@
+#!/bin/bash
+# Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the
+# License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+function main() {
+  ## Optional command to install the DCGM Diagnostic tool
+  # install_dcgmi
+
+  ## Retrieve the instance type
+  local instance_type
+  set_instance_type
+  fast_success_exit "$?" "Skipping execution of the GPU Health Check"
+
+  ## Check if the instance has an NVIDIA acceleration
+  is_nvidia
+  fast_success_exit "$?" "The GPU Health Check is running in an instance without GPU: skipping its execution."
+
+  ## Check if there are GPU associated to the job
+  if [ -z "$GPU_DEVICE_ORDINAL" ]; then
+#    log_info "The GPU Health Check is not required since there are no GPU associated to the job."
+#    return 0
+    nvidia_smi_output=$(nvidia-smi -L)
+    fast_success_exit "$?" "The GPU Health Check is running in an instance without nvidia-smi available: skipping its execution."
+    GPU_DEVICE_ORDINAL=$(echo "$nvidia_smi_output" | while IFS= read -r gpu ; do echo $gpu | cut -d ':' -f1 | sed s/"GPU "//; done | tr '\n' ',' | sed 's/.$//')
```

Comment:
please add a comment with an example of input and output of the transformation command